src/cljc/ataru/util.cljc

```diff
@@ -68,3 +69,16 @@
     (->> dropdown-options
         (filter (comp (partial = value) :value))
       (mapcat :followups))))
+
+(def ^:private b-limit 1024)
+(def ^:private kb-limit 102400)
+
+(defn size-bytes->str [bytes]
+  #?(:cljs (condp > bytes
+             b-limit (str bytes "B")
+             kb-limit (gstring/format "%.01fkB" (/ bytes 1024))
```

Comment:
Is it possible to extract the format fn (`gstring/format` or `clojure.core/format`) using the `#?` reader conditional (`(defonce format-fn #?(:cljs gstring/format :clj format))`) to make the `size-bytes->str` a bit simpler?